panda/src/express/pointerToArray.h

```diff
@@ -179,7 +179,7 @@ class PointerToArray : public PointerToArrayBase<Element> {
   INLINE void erase(iterator position);
   INLINE void erase(iterator first, iterator last);
 
-#if !defined(WIN32_VC) && !defined (WIN64_VC)
+#ifndef _WIN32
```

Comment:
Should be `_MSC_VER` (and should check to see if it's still necessary)

Ditto the two edits below in this same file